### PR TITLE
Release v2.0.0-beta

### PR DESCRIPTION
**Pull request by kevinoo**
This release support Chartjs v.3.x.x-beta10

### CODE REVIEW

**Comment by kevinoo** [comment]
Released version to support ChartJS v3.x   😄

**Comment by csholmq** [comment]
> Released version to support ChartJS v3.x 😄

Just tested this with Chart.js v3.5.1 but all I got was 
```
chartjs-plugin-streaming.min.js:formatted:388
Uncaught TypeError: Cannot read properties of undefined (reading 'type')
```
Is this incompatible with the newest version?

File: README.md [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
 ## Installation
 
-You can download the latest version of chartjs-plugin-streaming from the [GitHub releases](https://github.com/nagix/chartjs-plugin-streaming/releases/latest).
+You can download the latest version of @kevinoo/chartjs-plugin-streaming from the [GitHub releases](https://github.com/kevinoo/chartjs-plugin-streaming/releases/latest).
 
 To install via npm:
 
 ```bash
-npm install chartjs-plugin-streaming --save
+npm install @kevinoo91/chartjs-plugin-streaming --save
```
Comment (benmccann) [inline README.md:22]: these changes shouldn't be checked into this repo

File: samples/line-horizontal.html [hunk 2]
```diff
@@ -86,30 +87,35 @@
...
 				},
-				tooltips: {
-					mode: 'nearest',
-					intersect: false
-				},
-				hover: {
-					mode: 'nearest',
-					intersect: false
-				}
+				// tooltips: {
```
Comment (benmccann) [inline samples/line-horizontal.html:111]: we shouldn't have commented out code. we should fix it, remove it if no longer applicable, or at least add a comment explaining why it's commented out and what would need to be done to address it

**Review by benmccann — COMMENTED** [review]
Does this PR still do anything? `master` is already using Chart.js 3: https://github.com/nagix/chartjs-plugin-streaming/blob/ab4bc9ea05b638a613807dc3e4632aa825a55373/package.json#L57

I wonder if this PR should be closed? Or at least rebased...

**Comment by kevinoo** [comment]
Dear benmccann,
the PR is old  😁  I am going to close it  😊

**Pull request closed without merging**